addon/components/basic-dropdown/content.js

```diff
@@ -47,7 +59,15 @@ export default WormholeComponent.extend({
   willDestroyElement() {
     this._super(...arguments);
     let dropdown = self.window.document.getElementById(this.get('dropdownId'));
-    this.get('appRoot').removeEventListener('mousedown', this.handleRootMouseDown, true);
+    const appRoot = this.get('appRoot');
```

Comment:
Use let. I will add `ember-suave` soon and it will compain about const being used outside the top-level scope.